example/main.go

```diff
@@ -1,16 +1,18 @@
 package main
+
 // Copyright © 2015 Ian Denhardt <ian@zenhack.net>
 // Licensed under the GPLv3 or later.
 // See COPYING at the root of the repository for details.
 
 import (
-	".."
 	"flag"
 	"fmt"
+
+	"github.com/kalbasit/go.notmuch"
```

Comment:
I think it makes the most sense to leave this as a relative import; making it absolute like this means the example will run against one person's fork, rather than the source tree it's supposed to be an example for.